### PR TITLE
[circle2circle] Add fuse_rmsnorm option

### PR DESCRIPTION
**Pull request by seockho-kim**
This commit adds 'fuse_rmsnorm' option to circle2circle.

ONE-DCO-1.0-Signed-off-by: Seockho Kim seockho.kim@samsung.com

issue: https://github.com/Samsung/ONE/issues/13964
draft: https://github.com/Samsung/ONE/pull/13967

### CODE REVIEW

File: compiler/circle2circle/src/Circle2Circle.cpp [hunk 1]
```diff
@@ -301,6 +302,7 @@ int entry(int argc, char **argv)
   option_str_to_enum["fuse_preactivation_batchnorm"] = Algorithms::FusePreActivationBatchNorm;
   option_str_to_enum["fuse_prelu"] = Algorithms::FusePRelu;
   option_str_to_enum["fuse_gelu"] = Algorithms::FuseGelu;
+  option_str_to_enum["fuse_rmsnorm"] = Algorithms::FuseRmsNorm;
```
Comment (seanshpark) [inline compiler/circle2circle/src/Circle2Circle.cpp:305]: plz add setting flag from luci/pass if this option is enabled.

Comment (seockho-kim) [reply compiler/circle2circle/src/Circle2Circle.cpp:305]: > plz add setting flag from luci/pass if this option is enabled.

Where's setting flag in luci pass? I couldn't understand it.

Comment (seanshpark) [reply compiler/circle2circle/src/Circle2Circle.cpp:305]: Ah, there was refactoring and has changed ... -_-;

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**